src/lib/import/import-utils.ts

```diff
@@ -113,7 +113,17 @@ export class ImportUtils {
                 definition += node.nodeValue?.trim();
             }
         }
-        return definition;
+        return this.removeExcessiveIndents(definition);
+    }
+
+    public removeExcessiveIndents(action: string): string {
+        action = action.trim().replace(/\t/g, '    ');
+        const lines = action.split('\n');
+        let commonIndent = Math.min.apply(lines.map(l => l.length - l.trimStart().length));
```

Comment:
the documentation of the `apply` method implies that the method should get two arguments.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply

Why use the `apply` method in the first place and not just pass the result of the `map` method to the `min` method via the spread operator? `Math.min(...lines.map(    ))`

Reply:
stackoverflow

Reply:
If you are satisfied with the author's answer, please resolve the conversation.